backend/src/main/java/com/soen341/instagram/service/impl/PictureService.java

```diff
@@ -22,20 +26,23 @@
 import java.nio.file.Paths;
 import java.util.Date;
 import java.util.Optional;
+import java.util.Set;
 import java.util.List;
 import java.util.stream.Collectors;
 
 @Service("pictureService")
 public class PictureService {
 
+	@Autowired
+	private AccountRepository accountRepository;
```

Comment:
This isn't used. Please remove.

Reply:
accountRepository is used by getCurrentUser()